distribution/lib/Standard/Base/0.0.0-dev/src/Polyglot.enso

```diff
@@ -15,6 +15,13 @@ type Polyglot
 get_array_size : Any -> Integer
 get_array_size array = @Builtin_Method "Polyglot.get_array_size"
 
+## Reads the element in a given polyglot array object.
+
+   Arguments:
+   - index: The index to get the element from.
+read_array_element : Any -> Integer -> Any
```

Comment:
After [Interop for Array & Vector](https://www.pivotaltracker.com/n/projects/2539304/stories/183000876) we shall not need any special `read_array_element` method. One would invoke `Array.at` on any polyglot array directly.